### PR TITLE
Add new Skin API

### PR DESCRIPTION
**Pull request by Creeperz653**
Added creepernation skin api

### CODE REVIEW

**Comment by onebeastchris** [comment]
just out of curiosity, what makes it better then the existing tydium api?

**Comment by Creeperz653** [comment]
> just out of curiosity, what makes it better then the existing tydium api?

I’m not going to say better because every api has its own thing but we are going to have feature never seen before this api is not the same I’d suggest you talk to blue @bluetree242

**Comment by Creeperz653** [comment]
Our up time is doing well incase your wondering  https://hetrixtools.com/r/d1539367bbd131746b9d387c118e0f5b/ tracked since 21 september (excluding geyser api)

**Comment by SocMinarch** [comment]
If you’re going to propose a new API, it’s not upstreams job to chase someone else on what it does that the existing one doesn’t already do.

**Comment by BlueTree242** [comment]
Both APIs existed for a while, it's not new

**Comment by SocMinarch** [comment]
It’s new to the project

**Comment by SocMinarch** [comment]
And unless you had permission from both GeyserMC and Tydium, I really wouldn’t recommend setting up uptime monitors for their endpoints.

**Comment by Creeperz653** [comment]
> It’s new to the project

No not really it was private for awhile I mainly used it for my server

**Comment by Creeperz653** [comment]
> If you’re going to propose a new API, it’s not upstreams job to chase someone else on what it does that the existing one doesn’t already do.

tydiumcrafts api does it’s own thing my api has its own set of features to make it different from the others

**Comment by SocMinarch** [comment]
> > If you’re going to propose a new API, it’s not upstreams job to chase someone else on what it does that the existing one doesn’t already do.
> 
> tydiumcrafts api does it’s own thing my api has its own set of features to make it different from the others

And they are?

**Comment by Creeperz653** [comment]
> > > If you’re going to propose a new API, it’s not upstreams job to chase someone else on what it does that the existing one doesn’t already do.
> > 
> > 
> > tydiumcrafts api does it’s own thing my api has its own set of features to make it different from the others
> 
> And they are?

I’ve said it in the repo but we have more private features we aren’t finished coding but one of them will be Catcha another will hopefully solve an issue that fixed it so the apis not only for geyser I can’t revealed anything more

**Comment by Creeperz653** [comment]
I’ve got to go for now but I’ll be back

**Comment by Tydium** [comment]
> And unless you had permission from both GeyserMC and Tydium, I really wouldn’t recommend setting up uptime monitors for their endpoints.

Yeah I never really even gave permission for that lol…

**Comment by Jens-Co** [comment]
With this I'll lock the conversation. project leads will review this and merge it upon approval.

File: _docs/other/community-geyser-projects.md [hunk 1]
```diff
@@ -38,6 +38,15 @@ A Spigot, BungeeCord, and Velocity plugin to create custom Bedrock Edition forms
 
 Creators: [Konicai](https://github.com/Konicai), [Jens](https://github.com/Jens-Co)
 
+## CreeperNation Skin API
+
+An API to show java or floodgate bedrock skins depending on uuid or username
```
Comment (Chew) [inline _docs/other/community-geyser-projects.md:43]:
```suggestion
An API to show Java or Bedrock skins depending on UUID or username
```

File: _docs/other/community-geyser-projects.md [hunk 2]
```diff
@@ -38,6 +38,15 @@ A Spigot, BungeeCord, and Velocity plugin to create custom Bedrock Edition forms
...
+
+* [Website](https://api-docs.creepernation.xyz)
+
+Creators: [creeperz653](https://github.com/creeperz653), [BlueTree242](https://github.com/BlueTree242)
+
```
Comment (Chew) [inline _docs/other/community-geyser-projects.md:48]:
```suggestion
```

**Review by RednedEpic — APPROVED** [review]

**Review by Chew — APPROVED** [review]

**Review by jackson-57 — APPROVED** [review]

**Pull request merged**